.konflux/Dockerfile

```
@@ -1,35 +1,22 @@
+# See README.Konflux.md before editing this Dockerfile
+
 # build stage
 FROM registry.redhat.io/rhel9-4-els/rhel:9.4-1044 AS build-image
 WORKDIR app
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)